### PR TITLE
add admin questions

### PR DESCRIPTION
**Pull request by garysassano**
## PR Type

- [x] Adding new question(s)
> **Warning**: We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.
- [ ] Other content changes (updating questions, answers, explanations or study resources)
- [ ] Code changes (non-content)
- [ ] Documentation changes
- [ ] Other

### CODE REVIEW

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
Few small things except for the question 13 which I am not a fan of. Please take a look at that one again

File: content/questions/admin/question-013.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+question: "What are the steps to enable and enforce SAML SSO for a single organization?"
+archetype: "questions"
+title: "Question 013"
+draft: false
+---
+
+> https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/enabling-and-testing-saml-single-sign-on-for-your-organization#enabling-and-testing-saml-single-sign-on-for-your-organization
+1. [x] In the main menu, select `Organization`, then `Security Settings`, enable `SAML SSO`, fill in the IdP details, test the SSO configuration, and then enforce SAML SSO.
```
[the remark this answers is not in the document]
Comment (garysassano) [reply content/questions/admin/question-013.md:9]: What do you mean? That's just a list of the steps as mentioned in the link. Would you prefer to have them as breadcrumbs joined by `>` character?

File: content/questions/admin/question-013.md
[the remark this answers is not in the document]
Comment (garysassano) [reply content/questions/admin/question-013.md:1]: Main menu is the one that appears when clicking on your profile picture.

File: content/questions/admin/question-013.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (FidelusAleksander) [reply content/questions/admin/question-013.md:9]: Where in the link these steps are mentioned? 

To me it looks like the third answer here is correct.. not the first one

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]

File: content/questions/admin/question-013.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (garysassano) [reply content/questions/admin/question-013.md:9]: You are right, I put one of the wrong options as the solution.

**Comment by FidelusAleksander** [comment]
LMK once you fix pre-commit and I will merge this.

Please also add the missing words that are raising warnings in https://github.com/FidelusAleksander/ghcertified/blob/master/cspell.json

"SCIM"
"Keycloak"
"Entra"

**Review by FidelusAleksander — APPROVED** [review]
LGTM! 

Thanks for adding this 🚀

**Pull request merged**